selftests/functional/test_statuses.py

```diff
@@ -25,8 +25,6 @@
 #                     [msg_in, ...])
 EXPECTED_RESULTS = {'skip-setup-d304': ('SKIP',
                                         ['setup pre',
-                                         'teardown pre',
-                                         'teardown post',
```

Comment:
Yeah, I should have noticed this as not-expected behavior, but it sounded right when I was merging it...

Reply:
I mean it shouldn't really matter as `tearDown` should be written the way it does not require `setUp` to be executed, but this version better sticks to what we define as SKIP.